logical/src/main/java/org/apache/drill/common/logical/StoragePluginConfig.java

```diff
@@ -19,11 +19,9 @@
 
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-@JsonInclude(JsonInclude.Include.NON_DEFAULT)
```

Comment:
Please revert these changes. It helps to preserve backward compatibility for the case when the new option was added, the user started a newer version of Drill, but after that decided to use previous one.

Reply:
Sure, this is for fixing one ut case, I have tried another way.